paasta_tools/api/api.py

```diff
@@ -118,6 +119,8 @@ def setup_paasta_api():
         cached=False,
     )
 
+    settings.kubernetes_clients = kubernetes_tools.get_kubernetes_clients()
```

Comment:
Can this fail? Maybe add a try except as we can't be sure the config etc exists?